.drone.yml

```diff
@@ -77,6 +77,26 @@ steps:
     event:
     - push
 
+- name: docker-mirror-istio
+  image: plugins/docker
+  environment:
+    PASSWORD:
+      from_secret: docker_password
+    USERNAME:
+      from_secret: docker_username
+  commands:
+  - docker login -u $USERNAME -p $PASSWORD
```

Comment:
I think you need to use DinD image or dapper even. The plugin entrypoint is the plugin-binary.

Reply:
we did something similar here that seems to have worked: https://github.com/rancher/ingress-nginx/blob/46c70a1f6919537b25affa642ffbec6db10292be/.drone.yml#L20

Not sure if mounting in the docker socket was required.

Reply:
The image you are using here is plugins/docker that uses library docker
https://github.com/rancher/ingress-nginx/blob/46c70a1f6919537b25affa642ffbec6db10292be/.drone.yml#L21
so it brings the CLI uses the host docker socket.

Reply:
ah, yes. alright let me switch over to that

Reply:
Volume mount for the host socket

Reply:
Nevermind